packages/edit-site/src/components/page-pages/index.js

```diff
@@ -131,13 +138,16 @@ export default function PagePages() {
 						</a>
 					);
 				},
+				elements: authors,
+				enableColumnFiltering: true,
```

Comment:
Sometimes it's "enableColumnFiltering" and sometimes it's "enableColumnFilter" what's the difference?

Reply:
A leftover from previous experiments.